### ISSUE TITLE
[Feature] Support different meta info stored in different namespace (when use kubeapi )

### ISSUE THREAD

**Issue by oeular** [kubevela/kubevela#3853]
- At present, many resources are stored in configmaps (when use kubeapi), such as application, component, env, target, env-binding, and etc.
- BUT they are in the SAME namespace, it will be more hard to manage. 
- So, different resources at best can be stored in different namespace (when use kubeapi), which will be clearer and more convenient to manage.

**Comment by FogDong** [comment]
There're some resources that don't have namespace like user info, etc.
Should these data all stored in some default namespace?

/cc @barnettZQG

**Comment by fourierr** [comment]
The meta resource was supported to store in different ns, but just the configration variable was not exposed to env or arguments

**Comment by StevenLeiZhang** [comment]
I don't think it is reasonable to distribute meta info to different namespace。 What is the disadvantage of centralized store?

**Comment by oeular** [comment]
@StevenLeiZhang  
As we own more and more applications, it will be more difficult to manage the related resources for they are all in one namespace. 
And on the side of K8s，the number of the resoruces in one namespace will low down the efficiency of the resources creation and updating.

**Comment by barnettZQG** [comment]
@oeular The DB driver with Configmap is a sample implement, for a production platform that manages many applications we recommend using MongoDB or a planned MySQL driver.

**Comment by briankane** [comment]
This issue has been inactive for over 2 years and is scheduled for closure as part of our backlog maintenance.

  **If this issue is still relevant:**
  - Please comment below to keep it open
  - Provide updated context if the original description is outdated

  **Timeline:** If there's no activity by **Friday, October 24th**, this issue will be automatically closed.

  **Note:** Issues can always be reopened if they become relevant again. We encourage the community to reopen and reprioritize any incorrectly 
  closed issues.

**Comment by briankane** [comment]
This issue has been closed following the stale issue notice. 

If this issue is still relevant and should remain open, please:
1. Reopen the issue
2. Remove the 'stale' label
3. Provide an update on the current status or importance

Thank you for your understanding and contribution to KubeVela!
